keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/TitusClusterHandler.kt

```diff
@@ -497,6 +497,9 @@ class TitusClusterHandler(
     val healthy: Boolean = activeServerGroups.all {
       it.instanceCounts?.isHealthy(resource.spec.deployWith.health) == true
     }
+
+    eventPublisher.publishEvent(ResourceHealthEvent(resource, healthy))
```

Comment:
Do we have documentation somewhere that describes the contract that we expect a resource handler to conform to? This PR looks like it modifies the existing contract by now expecting resource handlers to publish resource health events.

Reply:
No, I don't think we do.

Longer term I really want to create a common base for cluster handlers, because so much of the code is doing the same transforms. I think that would help alleviate the need for making the expectations clear.

Reply:
I tried to document the contract not too long ago: https://github.com/spinnaker/keel/blob/c3b40de05e19562fadd8ec3d6636e4f84f568e6e/keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/ResourceHandler.kt#L16-L38